tests/data/small_sky_object_catalog/properties

```diff
@@ -2,13 +2,13 @@
 obs_collection=small_sky_object_catalog
 dataproduct_type=object
 hats_nrows=131
-hats_col_j2000_ra=ra
-hats_col_j2000_dec=dec
+hats_col_ra=ra
+hats_col_dec=dec
 hats_max_rows=1000000
 hats_order=0
 moc_sky_fraction=0.08333
-hats_builder=hats-import v0.3.6.dev22+g6346e4d
-hats_creation_date=2024-09-30T13\:52UTC
-hats_estsize=121
+hats_builder=hats-import v0.3.6.dev26+g40366b4
+hats_creation_date=2024-10-11T15\:02UTC
+hats_estsize=74
```

Comment:
Is this estimated size smaller because it's now only accounting for the parquet files?

Reply:
I suspect that it's due to hats PR [#361](https://github.com/astronomy-commons/hats/pull/361) - this switched the pointmap.fits file to store dtype=np.int32 instead of int64. which would cut the size of that file by about 50%, and most of the storage for very-small-catalogs is taken up by that one file.